test/arithmetic_test.cc

```diff
@@ -8,26 +10,26 @@ class MulUnsignedHiTest : public testing::Test {
   void TearDown() override {}
 };
 
-using rv64_emulator::libs::arithmetic::MulUnsignedHi;
+using rv64_emulator::libs::arithmetic::MulUint64Hi;
+using rv64_emulator::libs::arithmetic::PortableMulUnsignedHi;
 
 TEST_F(MulUnsignedHiTest, MulUnsignedHiUint16) {
-  ASSERT_EQ(MulUnsignedHi<uint16_t>(0x1234, 0x5678), 0x0626);
+  ASSERT_EQ(PortableMulUnsignedHi<uint16_t>(0x1234, 0x5678), 0x0626);
 }
 
 TEST_F(MulUnsignedHiTest, MulUnsignedHiUint32) {
-  ASSERT_EQ(MulUnsignedHi<uint32_t>(0x12345678, 0x9ABCDEF0), 0x0b00ea4e);
+  ASSERT_EQ(PortableMulUnsignedHi<uint32_t>(0x12345678, 0x9ABCDEF0),
+            0x0b00ea4e);
 }
 
 TEST_F(MulUnsignedHiTest, MulUnsignedHiUint64) {
-  using T = uint64_t;
-
-  T a = 0x123456789ABCDEF0;
-  ASSERT_EQ(MulUnsignedHi<T>(a, a), 0x014B66DC33F6ACDC);
+  uint64_t a = 0x123456789ABCDEF0;
```

Comment:
Variable names are meaningless

Reply:
That's because `a` is the oprand of "mul" instruction. And it's easy to derive the meaning of `a` in the short and brief code context.